pkg/server/backuprestoreserver.go

```diff
@@ -88,6 +91,35 @@ func NewBackupRestoreServer(logger *logrus.Logger, config *BackupRestoreComponen
 
 // Run starts the backup restore server.
 func (b *BackupRestoreServer) Run(ctx context.Context) error {
+	var inputFileName string
+	var err error
+
+	// (For testing purpose) If no ETCD_CONF variable set as environment variable, then consider backup-restore server is not used for tests.
+	// For tests or to run backup-restore server as standalone, user needs to set ETCD_CONF variable with proper location of ETCD config yaml
+	etcdConfigForTest := os.Getenv("ETCD_CONF")
+	if etcdConfigForTest != "" {
+		inputFileName = etcdConfigForTest
+	} else {
+		inputFileName = etcdConfig
+	}
+
+	configYML, err := os.ReadFile(inputFileName)
+	if err != nil {
+		b.logger.Fatalf("Unable to read etcd config file: %v", err)
+		return err
+	}
+
+	config := map[string]interface{}{}
+	if err := yaml.Unmarshal([]byte(configYML), &config); err != nil {
+		b.logger.Fatalf("Unable to unmarshal etcd config yaml file: %v", err)
+		return err
+	}
+
+	initialClusterMap, err := types.NewURLsMap(fmt.Sprint(config["initial-cluster"]))
+	if err != nil {
+		b.logger.Fatal("Please provide initial cluster value for embedded ETCD")
```

Comment:
Please add a **release note** that this a `initial-cluster` field is now expected in the config.

Reply:
Done